L42-metadata-flags.md

```diff
@@ -0,0 +1,140 @@
+Utilize Metadata Flags -- Adding "Wait For Ready" Semantics
+----
+* Author(s): lidizheng
+* Approver: a11r
+* Status: Draft
+* Implemented in: Python
+* Last updated: October 16, 2018
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/Q6fgbn0RZLs
+
+## Abstract
+
+Enable gRPC Python client to utilize the “Wait For Ready” mechanism provided by C-Core, which underlying is to utilize the initial metadata flags. This mechanism can later be used to support future metadata flags.
```

Comment:
Suggestion to rewrite "Surface 'wait-for-ready' configuration in gRPC Python client that will use the mechanism provided by C-Core via the initial metadata flags. The implementation should allow adding future such configurations in gRPC Python easily.